packages/types/src/clerk.ts

```diff
@@ -909,6 +909,23 @@ export type UserButtonProps = UserButtonProfileMode & {
    * Controls the default state of the UserButton
    */
   defaultOpen?: boolean;
+
+  /**
+   * If true the UserButton will only render the popover.
+   * Enables developers to implement a custom dialog.
+   * @experimental This API is experimental and may change at any moment.
+   * @default false
+   */
+  __experimental_standalone?: boolean;
+
+  /**
+   * Notifies the caller that it's safe to unmount UserButton.
+   * It only fires when used in conjunction with `__experimental_standalone`.
+   * @experimental This API is experimental and may change at any moment.
+   * @default undefined
+   */
```

Comment:
What is the behavior when the caller doesn't close their popover implementation when this function is called? Do we continue rendering something so it doesn't look broken?

Reply:
It will continue to display that same standalone component.